user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Version.md

```diff
@@ -0,0 +1,65 @@
+---
+uid: Tutorial_Register_Catalog_Version
+---
+
+# Registering a new version of a connector in the Catalog
+
+This tutorial demonstrates how to add a new version to a catalog item using the [Catalog API](xref:Register_Catalog_Item).  
+We will be registering our own version of the following example [connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom), so go ahead and download the solution.
+
+## Prerequisites
+
+- Organization key with permission "Register catalog items".
+- A registered catalog item, see [register a catalog item tutorial](xref:Tutorial_Register_Catalog_Item)
+
+## Overview
+
+- [Step 1: Register catalog version URL](#step-1-register-catalog-version-url)
+- [Step 2: Authentication header](#step-2-authentication-header)
+- [Step 3: Register catalog version body](#step-3-register-catalog-version-body)
+- [Step 4: Register](#step-4-register)
+
+## Step 1: Register Catalog version URL
+
+In order to register our first version, we will use the catalog API Register version call.
+Create a new Http request using the POST http method and url  <https://api.dataminer.services/api/key-catalog/v1-0/catalog/{catalogid}/version/register>
+
+![Register version http url](~/user-guide/images/tutorial_catalog_registration_version_url.png)
+
+>[!note]
+Make sure you use the same id as you used for the [catalog item registration](xref:Tutorial_Register_Catalog_Item).
```

Comment:
```suggestion
Make sure you use the correct Catalog ID in the URL. If you followed the tutorial to [register a catalog item](xref:Tutorial_Register_Catalog_Item), this was returned at the end.
```

Reply:
its actually generated manually by them at the start, the return in the response is just a confirmation (api no longer generates a GUID)

Reply:
Ok, but if you word it like this you basically hard-link this tutorial to the one of the register item tutorial. It's perfectly possible they follow this tutorial with their own registered item, not the one from the other tutorial right?
I'd rephrase it slightly to make it clear it just has to be the correct ID

Reply:
aligned it here with the tutorial of github action

Reply:
ok